lib/components/viewers/trip-viewer.js

```diff
@@ -1,16 +1,17 @@
+/* eslint-disable jsx-a11y/label-has-for */
```

Comment:
Add FIXME

Reply:
Added in https://github.com/opentripplanner/otp-react-redux/pull/390/commits/11773ca2755c074d3ca83db0e40a92031c737b97.